base_layer/core/src/fee.rs

```diff
@@ -46,4 +44,9 @@ impl Fee {
             fee
         }
     }
+
+    /// Calculate the weight of a transaction based on the number of inputs and outputs
+    pub fn calculate_weight(num_inputs: usize, num_outputs: usize) -> u64 {
+        COST_PER_INPUT * num_inputs as u64 + COST_PER_OUTPUT * num_outputs as u64
+    }
```

Comment:
Nice. WEIGHT_PER_xxxPUT is a better name now, I guess.

Reply:
Function name changed to `weight_per_xxxput`, it still feels like there should be a better name.

Reply:
Lol. No, I meant

pub fn calculate_weight(num_inputs: usize, num_outputs: usize) -> u64 {
        WEIGHT_PER_INPUT * num_inputs as u64 + WEIGHT_PER_OUTPUT * num_outputs as u64
    }

Reply:
The original name was fine

Reply:
Oops, change applied correctly.